### PR TITLE
chore: applying integration202008 branch to qa-jcoin for August release

### PR DESCRIPTION
**Pull request by atharvar28**

### CODE REVIEW

File: qa-covid19.planx-pla.net/manifest.json [hunk 1]
```diff
@@ -4,27 +4,27 @@
...
-    "indexd": "quay.io/cdis/indexd:2020.07",
-    "peregrine": "quay.io/cdis/peregrine:2020.07",
-    "pidgin": "quay.io/cdis/pidgin:2020.07",
-    "revproxy": "quay.io/cdis/nginx:2020.07",
+    "fence": "quay.io/cdis/fence:integration202008",
+    "indexd": "quay.io/cdis/indexd:integration202008",
+    "peregrine": "quay.io/cdis/peregrine:integration202008",
+    "pidgin": "quay.io/cdis/pidgin:integration202008",
+    "revproxy": "quay.io/cdis/nginx:integration202008",
     "sheepdog": "quay.io/cdis/sheepdog:4.1.1",
```
Comment (paulineribeyre) [inline qa-covid19.planx-pla.net/manifest.json:16]: sheepdog should also be included

File: qa-covid19.planx-pla.net/manifest.json [hunk 2]
```diff
@@ -4,27 +4,27 @@
     "That's all I have to say"
   ],
   "versions": {
```
Comment (paulineribeyre) [inline qa-covid19.planx-pla.net/manifest.json:6]: the title says jcoin but this is covid19

**Comment by paulineribeyre** [comment]
@themarcelor @haraprasadj please check changes more carefully before approving PRs

File: qa-covid19.planx-pla.net/manifest.json (hunk 2 above)
Comment (atharvar28) [reply qa-covid19.planx-pla.net/manifest.json:6]: dam yes!

Comment (haraprasadj) [reply qa-covid19.planx-pla.net/manifest.json:6]: 🤦 ... sorry!

**Comment by atharvar28** [comment]
Wrong env!

**Pull request closed without merging**